### PR TITLE
[Edge Extensions] Publish/Update extension documentation changes for the new flow.

### PR DESCRIPTION
**Pull request by ShashankMG123**
We have recently modified the flow for publishing and updating an extension on Edge.
This PR updates the documentation to address these new changes.

Dev Contact: shashankmg@microsoft.com
PM : ashish.poddar@microsoft.com

For a walkthrough of the new flow, credentials to the test account please drop a mail the the above 2 aliases.

**Rendered articles for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/publish-extension?branch=pr-en-us-1774
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/update-extension?branch=pr-en-us-1774

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Go to the [developer dashboard](https://partner.microsoft.com/dashboard/microsof
 
 Use the **Packages** webpage to upload the zip file of your extension package.  You can only upload one package at a time.  Your submission is blocked if the package upload isn't successful on the **Packages** webpage.
 
-To upload the package, select and drag the package onto the upload field.  Also, you can select **Browse your files**.  After the package is uploaded, your package is validated.  After validation succeeds, review the extension details, and then select **Next** to continue.  If there are validation errors, resolve the issues and try uploading again.
+To upload the package, select and drag the package onto the upload field.  Also, you can select **Browse your files**.  After the package is uploaded, your package is validated.  After validation succeeds, review the extension details, and then select **Continue** to proceed.  If there are validation errors, resolve the issues and try uploading again.
```
Comment (vchapel) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:76]: @ShashankMG123  I think some of this section needs to be numbered steps. I am also not sure about the button / link names. If you could provide raw screenshots at the "Save & Continue" steps at line 76, 100, and 117, I can verify the UI and possibly include one or more images as appropriate.

Comment (ShashankMG123) [reply microsoft-edge/extensions-chromium/publish/publish-extension.md:76]: Changes:
1. 
image
This has been changed from "Next" to "Continue"
2.
image
Notice the buttons on the command bar. The "Save & Continue" button was previously a "Save draft" button

Brief about the new flow:
1. Previously there were situations where the Publish button was disabled even if all required data was filled (due to multiple reasons for example if there was a publish in progress) and we did receive feedback where the user did not understand why the button was disabled. In the new flow the publish button will be enabled as long as all required fields are completed. If there are any reason which prevents the user from publishing we will be surfacing the reason post the button click.
2. We have also made it easier for the user to publish an update by just clicking the publish button (previously the requirement was the click save draft and then publish.)

Let me know if this requires some sort of demo from my end to make the flow more clear.

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 2]
```diff
@@ -71,9 +71,15 @@ Go to the [developer dashboard](https://partner.microsoft.com/dashboard/microsof
 <!-- ========================================================================== -->
 ## Step 2:  Upload the extension package
 
-Use the **Packages** webpage to upload the zip file of your extension package.  You can only upload one package at a time.  Your submission is blocked if the package upload isn't successful on the **Packages** webpage.
+1. Go to the **Packages** webpage to upload the zip file of your extension package.
 
-To upload the package, select and drag the package onto the upload field.  Also, you can select **Browse your files**.  After the package is uploaded, your package is validated.  After validation succeeds, review the extension details, and then select **Next** to continue.  If there are validation errors, resolve the issues and try uploading again.
+   You can only upload one package at a time. Your submission is blocked if the package upload isn't successful on the **Packages** webpage.
+
+1. To upload your package, drag and drop the zip file to the **Upload** section of the webpage. Or you can select **Browse your files** to open a dialog box to select the package.
```
Comment (vchapel) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:78]: @ShashankMG123 please check the user interface terms in bold.

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 3]
```diff
@@ -71,9 +71,15 @@ Go to the [developer dashboard](https://partner.microsoft.com/dashboard/microsof
...
+
+   After your package is uploaded, it is validated. If there is a validation error, resolve the issue and try uploading your package again.
+
+1. When validation succeeds, review the extension details, and then select **Continue** to proceed.
```
Comment (vchapel) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:82]: @ShashankMG123 please check the user interface terms in bold.

File: microsoft-edge/extensions-chromium/publish/publish-extension.md (hunk 2 above)
Comment (ShashankMG123) [reply microsoft-edge/extensions-chromium/publish/publish-extension.md:78]: Not sure if we can call it the Upload section
image

Comment (vchapel) [reply microsoft-edge/extensions-chromium/publish/publish-extension.md:78]: Fixed

**Review by vchapel — APPROVED** [review]
LGTM

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**